### PR TITLE
Update submission UI element finding

### PR DESCRIPTION
**Pull request by vanwagonet**
For all the Submission tests, use ElementWrapper instead of UITestElement

refs: none
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 93.88% | 93.88% | 0%
**Student** | 92.51% | 92.51% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.71% | 92.71% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/StudentUITests/Submissions/SubmissionDetails/SubmissionDetailsTests.swift [hunk 1]
```diff
@@ -57,14 +57,13 @@ class SubmissionDetailsTests: StudentTest {
...
-        page.assertEnabled(.attemptPickerToggle, false)
-        page.assertText(.attemptPickerToggle, equals: DateFormatter.localizedString(from: submittedAt, dateStyle: .medium, timeStyle: .short))
-        page.assertVisible(.onlineTextEntryWebView)
+        XCTAssertTrue(SubmissionDetails.onlineTextEntryWebView.waitToExist(Timeout(value: 5)))
+        XCTAssertFalse(SubmissionDetails.emptyView.isVisible)
+        XCTAssertFalse(SubmissionDetails.attemptPicker.isVisible)
+        XCTAssertTrue(SubmissionDetails.attemptPickerToggle.isVisible)
+        XCTAssertFalse(SubmissionDetails.attemptPickerToggle.isEnabled)
+        XCTAssertEqual(SubmissionDetails.attemptPickerToggle.label, DateFormatter.localizedString(from: submittedAt, dateStyle: .medium, timeStyle: .short))
+        XCTAssertTrue(SubmissionDetails.onlineTextEntryWebView.isVisible)
```
Comment (shichongrui) [inline Student/StudentUITests/Submissions/SubmissionDetails/SubmissionDetailsTests.swift]: I like this. Feels much more consistent with all of our other tests.

File: Student/StudentUITests/Tests/StudentTest.swift [hunk 2]
```diff
@@ -49,7 +49,10 @@ class StudentTest: XCTestCase {
         // This sleep helps ensure old views got cleaned up, so EG2 doesn't find them accidentally.
         sleep(1) // FIXME: Remove this and fix flakiness better.
 
-        getApp = { return DriverFactory.getXCUITestDriver(xcuiApp!, testCase: self) }
+        getApp = {
+            // return DriverFactory.getXCUITestDriver(xcuiApp!, testCase: self)
```
Comment (shichongrui) [inline Student/StudentUITests/Tests/StudentTest.swift]: Does this need to stick around?

Comment (vanwagonet) [reply Student/StudentUITests/Tests/StudentTest.swift]: We're going to be going back and forth a lot for now, so I kept it in as a comment. I tested both, and both passed.

**Pull request merged**